jwst/skymatch/skyimage.py

```diff
@@ -446,6 +446,12 @@ def calc_sky(self, overlap=None, delta=True):
 
                 # set pixels in 'fill_mask' that are inside a polygon to True:
                 x, y = self.wcs_inv(ra, dec)
+                for xi, yi in zip(x, y):
+                    print(f"{xi:.16g},   {yi:.16g")
+                xcheck = np.ndarray.round(x)
+                ycheck = np.ndarray.round(y)
```

Comment:
```suggestion
                xcheck = np.ndarray.round(x).astype(int)
                ycheck = np.ndarray.round(y).astype(int)
```
just to be paranoid about FLT differences.